src/heuristicblocking.js

```diff
@@ -211,6 +211,13 @@ HeuristicBlocker.prototype = {
       snitch_map.setItem(tracker_origin, firstParties);
     }
 
+    var action_map = this.storage.getBadgerStorageObject('action_map');
+    // If tracker_origin is not yet in action_map, add it with action `constants.ALLOW`
+    if (!action_map.hasItem(tracker_origin)) {
```

Comment:
No ... PB records tracking by origin in snitch map: "snitch_map is our collection of potential tracking base_domains. The key is a base domain (ETLD+1) ..."